### DISCUSSION TITLE
【共学营第一次】只显示一个交互的端口，没有另外三个，希望得到解答，谢谢

### DISCUSSION THREAD

**Discussion by SMARTJIANG2022** [NonceGeek/Move-Camp#267] [Q&A]
大家好，我在共学营move第一次的训练中，执行到1.4.3修改调用代码，执行到【定位到src/modal.jsx，修改!! KEY PLACE为相应的 fun】，有一些不理解，不知道如何进行下去
<img width="1036" alt="678447b38221307b6e878006799333b" src="https://user-images.githubusercontent.com/113282009/216611525-a0f4bda6-ee6c-4241-b542-e29a29e01782.png">

用前端进行交互也会出现这种情况。
<img width="745" alt="fc72b5d519ac263f16e89dfb7e9e1a6" src="https://user-images.githubusercontent.com/113282009/216609578-76ca4ad2-0e09-4795-a692-459ff2b0348c.png">
在localhost;3000中查看，只能看到一个交互的端口就是init_counter,没有其他另外三个端口。
<img width="745" alt="fc72b5d519ac263f16e89dfb7e9e1a6" src="https://user-images.githubusercontent.com/113282009/216612193-a9dd149d-3ba9-4d82-9856-bed53822a452.png">
请问是因为上述的【定位到src/modal.jsx，修改!! KEY PLACE为相应的 fun】没有修改成功的原因吗？

**Comment by xiegudong45** [comment]
可以看下1.4.5章节，其实就是需要你自己去实现Get Counter，Incr_counter和Incr_counter_by三个功能(实现前端的按钮 + 调用对应的合约func)。前端不知道怎么写的话，可以参考init_counter相关的代码
